controllers/categories.controller.js

```diff
@@ -0,0 +1,8 @@
+const { selectCategories } = require("../models/categories.model");
+
+exports.getCategories = (request, response) => {
+  //   console.log("Hi, I'm in controller");
+  selectCategories().then((categories) => {
+    response.status(200).send({ categories: categories });
+  });
+};
```

Comment:
Please remove all your console.logs. These shouldn't be in your code when your commit and push 🙂